### PR TITLE
Create azure_pim_role_assigned_outside_of_pim.yml

### PR DESCRIPTION
**Pull request by MarkMorow**
### Summary of the Pull Request

Alerts that are part of Privileged Identity Management (PIM) now have an API for interaction. There are 7 total alerts. Adding the first one to make sure it aligns to SIGMA rule standards. The first rule is to alert when a role is assigned outside of PIM. 

### Changelog

new: Privileged Role Assigned Outside of PIM

### Example Log Event

None

### Fixed Issues

None

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by MarkMorow** [comment]
@nasbench You will have to update the pipeline test to include this new log source for PIM alerts. Once this one is looking good I'll push the other 6 rules up. Thanks for all your help!

**Comment by nasbench** [comment]
Hey @MarkMorow thanks for this new batch. I update the logsource test and used `pim` as the name (as i think its fitting and shorter).

You can commit the other 6 rules so we can review them

**Comment by MarkMorow** [comment]
@nasbench thanks for the quick turn around. That should be all of them!

File: rules/cloud/azure/privileged_identity_management/azure_pim_too_many_global_admins.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Too Many Global Admins
+id: 7bbc309f-e2b1-4eb1-8369-131a367d67d3
+status: experimental
+description: Identifies when a too many global administrator assignments
```
Comment (nasbench) [inline rules/cloud/azure/privileged_identity_management/azure_pim_too_many_global_admins.yml:4]:
```suggestion
description: Identifies an event where there are there are too many accounts assigned the Global Administrator role.
```

**Review by nasbench — APPROVED** [review]

**Pull request merged**